HashMap.py

```diff
@@ -0,0 +1,61 @@
+class Node:
```

Comment:
Time and Space complexity?

Reply:
Time complexity: O(1)
Space complexity: O(1); no additional space apart from the array of linked lists